packages/core/stories/radio-checkbox.stories.js

```diff
@@ -44,6 +44,45 @@ storiesOf('Radio & Checkbox', module)
       </div>
     </React.Fragment>
   ))
+  .add('Radio pill group', () => (
+    <React.Fragment>
+      <div className="ray-radio-pill">
```

Comment:
Do you think there should be a wrapping element? Not sure 🤔

Reply:
consulted with design and we decided to have a wrapper so that this component can take up 100% of its container and grow/shrink accordingly